src/main.py

```diff
@@ -55,6 +56,25 @@
 api_logger.addHandler(stream_handler)
 logger.addHandler(stream_handler)
 
+# putting it here for the time being until frontend is refactored
+class ConfirmationView(discord.ui.View):

```

Comment:
Gonna be honest do not know enough about the discord apis to properly review this, but it seems to be non-blocking so Imma just trust you and if it doesn't work I'll rollback